tools/accuracy_checker/configs/human-pose-estimation-3d-0001.yml

```diff
@@ -0,0 +1,48 @@
+models:
+  - name: human-pose-estimation-3d-0001
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   intel/human-pose-estimation-3d-0001/FP32/human-pose-estimation-3d-0001.xml
+        weights: intel/human-pose-estimation-3d-0001/FP32/human-pose-estimation-3d-0001.bin
+        allow_reshape_input: True
+        adapter:
+          type: human_pose_estimation_3d
+          features_3d_out: features
+          keypoints_heatmap_out: heatmaps
+          part_affinity_fields_out: pafs
+
+      - framework: dlsdk
+        tags:
+          - FP16
+        model:   intel/human-pose-estimation-3d-0001/FP16/human-pose-estimation-3d-0001.xml
+        weights: intel/human-pose-estimation-3d-0001/FP16/human-pose-estimation-3d-0001.bin
+        allow_reshape_input: True
+        adapter:
+          type: human_pose_estimation_3d
+          features_3d_out: features
+          keypoints_heatmap_out: heatmaps
+          part_affinity_fields_out: pafs
+
+    datasets:
+      - name: cmu_panoptic_keypoints
+        data_source: cmu_panoptic_subset
```

Comment:
please move following strings to `dataset_definitions.yml`:
```yml
- name: cmu_panoptic_keypoints
  data_source: cmu_panoptic_subset
  annotation_conversion:
          converter: cmu_panoptic_keypoints
          data_dir: cmu_panoptic_subset
 annotation: cmu_panoptic_keypoints.pickle
```
and stay here only `- name: cmu_panoptic_keypoints`.
These changes are important for config reusage for validation on full dataset and light validation on 10 images.